content/404.html

```diff
@@ -6,25 +6,25 @@
 <meta name="description" content="">
 <meta name="keywords" content="">
 <title>Page Not Found | Hudi</title>
-<link rel="stylesheet" href="css/syntax.css">
+<link rel="stylesheet" href="/css/syntax.css">
```

Comment:
I noticed the "/" as well, locally.. Does it not break the css and js ? have you tried opening the generated content i.e `content/index.html`?

Reply:
I was looking into that too. Yes its is affecting css and js when I open the content/index.html. I see that this is coming from https://github.com/apache/incubator-hudi/pull/843  Let me see if there is a way to remove all the path occurrences and then update this PR again.

Reply:
@yanghua can you help with the removing the '/' changes in the original PR [#843](https://github.com/apache/incubator-hudi/pull/843)?

Reply:
@bhasudha It seems if the `href` does not start with `/`, then the URL of the resource can not be rendered correctly.

Reply:
@yanghua I find it to be opposite.. you can try opening `content/index.html` as a simple file inside the browser and it seems to break css/js .. Can you point @bhasudha may be to the line that changed this behavior.. we can test it out?

Reply:
@bhasudha @vinothchandar Sorry, I have tested. You are right. When I removed the start `/`, it worked fine. If it starts with `/`, the page can also be rendered correctly when run `bundle exec jekyll serve`, but it can not be rendered when we just open the generated HTML file with the browser.

So it's really an issue.

Reply:
@vinothchandar Should I create an issue to describe the problem and submit a PR?

Reply:
I ll let you and @bhasudha decide. If the fix is simple she can also do it in this PR if you give pointers

Reply:
OK @bhasudha You can fix it, just remove the first `/` and verify with the Jekyll to see if it works fine or not. Any problems, please let me know.